LICENSES-AND-NOTICES/SPECS/LICENSES-MAP.md

```diff
@@ -16,7 +16,7 @@ The Azure Linux SPEC files originated from a variety of sources with varying lic
+| OpenSUSE | Following [openSUSE guidelines](https://en.opensuse.org/openSUSE:Specfile_guidelines#Specfile_Licensing) | ant <br> ant-junit <br> antlr <br> aopalliance <br> apache-commons-beanutils <br> apache-commons-cli <br> apache-commons-codec <br> apache-commons-collections <br> apache-commons-collections4 <br> apache-commons-compress <br> apache-commons-daemon <br> apache-commons-dbcp <br> apache-commons-digester <br> apache-commons-httpclient <br> apache-commons-io <br> apache-commons-jexl <br> apache-commons-lang <br> apache-commons-lang3 <br> apache-commons-logging <br> apache-commons-net <br> apache-commons-pool <br> apache-commons-pool2 <br> apache-commons-validator <br> apache-commons-vfs2 <br> apache-parent <br> args4j <br> atinject <br> base64coder <br> bcel <br> bea-stax <br> beust-jcommander <br> bsf <br> byaccj <br> cal10n <br> cdparanoia <br> cglib <br> cni <br> containerized-data-importer <br> cpulimit <br> cri-o <br> ecj <br> fillup <br> flux <br> gd <br> geronimo-specs <br> glassfish-annotation-api <br> gnu-getopt <br> gnu-regexp <br> golang-packaging <br> guava <br> guava20 <br> hamcrest <br> hawtjni-runtime <br> httpcomponents-core <br> influx-cli <br> influxdb <br> jakarta-taglibs-standard <br> jansi <br> jarjar <br> java-cup <br> java-cup-bootstrap <br> javacc <br> javacc-bootstrap <br> javassist <br> jboss-interceptors-1.2-api <br> jdepend <br> jflex <br> jflex-bootstrap <br> jlex <br> jline <br> jna <br> jsch <br> jsoup <br> jsr-305 <br> jtidy <br> junit <br> junitperf <br> jzlib <br> kubevirt <br> kured <br> libcontainers-common <br> libtheora <br> libva <br> libvdpau <br> lynx <br> maven-parent <br> multus <br> objectweb-anttask <br> objectweb-asm <br> objenesis <br> oro <br> osgi-annotation <br> osgi-compendium <br> osgi-core <br> patterns-ceph-containers <br> plexus-classworlds <br> plexus-interpolation <br> plexus-pom <br> plexus-utils <br> proj <br> psl-make-dafsa <br> publicsuffix <br> qdox <br> regexp <br> relaxngDatatype <br> rhino <br> ripgrep <br> servletapi4 <br> servletapi5 <br> shapelib <br> slf4j <br> trilead-ssh2 <br> virtiofsd <br> xalan-j2 <br> xbean <br> xcursor-themes <br> xerces-j2 <br> xml-commons-apis <br> xml-commons-resolver <br> xmldb-api <br> xmlrpc-c <br> xmlunit <br> xpp2 <br> xpp3 <br> xz-java |
```

Comment:
@KavyaSree2610 there is currently a full 3.0 build/release underway with [a fix ](https://github.com/microsoft/azurelinux/pull/12886)for this license map issue included. This will cause a merge conflict on this PR delaying us post merging of 3.0 into fasttrack/3.0.

Can you remove this fix from this PR in favor of the one which will come in from 3.0 after the release? we will just ignore the license check error for now.